tests/Load/cleanup.sh

```
@@ -0,0 +1,89 @@
+    echo "S3 bucket $BUCKET_NAME deleted."
+else
+    echo "S3 bucket $BUCKET_NAME not found."
+fi
+
+echo "Terminating EC2 instances with tag: $INSTANCE_TAG"
+INSTANCE_IDS=$(aws ec2 describe-instances \
+  --filters "Name=tag:Name,Values=$INSTANCE_TAG" "Name=instance-state-name,Values=running" \
+  --query "Reservations[*].Instances[*].InstanceId" --output text --region "$REGION")
+
+if [ -n "$INSTANCE_IDS" ]; then
+    aws ec2 terminate-instances --instance-ids $INSTANCE_IDS --region "$REGION"
```

Shellcheck

[info] Double quote to prevent globbing and word splitting.

(SC2086)